add missing Issues content for Review List GA (#3255)
getsentry/sentry-docs: src/docs/product/error-monitoring/states-triage/index.mdx

---
title: Issue States and Triage
sidebar_order:
description: "Learn more about issue states and how to triage issues."
---

> An issue can be in one of 3 states:

Reviewer: ```suggestion
An issue can be in one of three states:
```

> - Unresolved: The default state for all new issues.
> - Ignored: The issue is ignored permanently or until a condition is met to move it to a state of Unignored. [Issue alerts](/product/alerts-notifications/issue-alerts/) are suppressed for ignored issues.
> - Resolved: The issue is resolved because it was fixed. You can also mark an issue as Resolved if you don't expect it to happen again.

Reviewer: ```suggestion
1. Unresolved: The default state for all new issues.
2. Ignored: The issue is ignored permanently or until a condition is met to move it to a state of Unignored. [Issue alerts](/product/alerts-notifications/issue-alerts/) are suppressed for ignored issues.
3. Resolved: The issue is resolved because it was fixed. You can also mark an issue as Resolved if you don't expect it to happen again.
```

> The For Review tab, or Review List, of the Issues page displays issues when any of the following happens:

Reviewer: ```suggestion
The "For Review" tab, or Review List, of the Issues page displays issues when any of the following happens:
```

TO DO Confirm meaning in first sentence; does someone have to manually do this reassigning to a team member after it gets assigned to a team?

### Link

Create an issue in an external project management tool like Jira that tracks the Sentry issue, or link an existing one. [Two-way sync](/product/sentry-basics/guides/getting-started/#33-issue-tracking) is available for some tools. Issues linked using [global integrations](/product/integrations/integration-platform/#public-integrations) or [internal integrations](/product/integrations/integration-platform/#internal-integrations) are searchable using the terms `is:linked` and `is:unlinked`.

### Mark Reviewed

Mark the issue as reviewed and [sentry.io](https://sentry.io) removes the reason label (the issue stays in the Unresolved state). Unlike ignoring an issue, this is a way to acknowledge an issue that you intend to fix at some point. This is only available on issues flagged for review)

Note that resolving or ignoring an issue implicitly marks the issue reviewed.

> To keep your Review List fresh, [sentry.io](https://sentry.io) automatically marks issues as reviewed if they remain untouched in the list for 7 days.

Reviewer: ```suggestion
To keep your Review List fresh, [sentry.io](https://sentry.io) automatically marks issues as reviewed if they remain untouched in the list for seven days.
```

### Delete

Delete an issue to remove it from the system. A new issue is created if a new instance of the issue (a new event) occurs.

### Delete and Discard

Delete and discard an issue to remove it from the system. A new issue is not created even if a new instance of the issue (a new event) occurs.

<Note>

The Auto Resolve setting defines the amount of time an issue can go without being seen again (that is, no new events on that issue) before it's automatically resolved. To check if this has been defined for a project, go to **[Project] > Settings > General Settings** and check the Event Settings section.

</Note>

While you can work to have zero unresolved issues ("Inbox Zero") by resolving or ignoring all issues, this usually isn't feasible for any non-trivial production application. However, it is feasible to have zero for-review issues by triaging the Review List. We recommend triaging these issues once a day to keep your issues manageable.

TO DO Figure out a home for these pieces of search information (if at all)
- `assigned:X` returns issues assigned to X

> - `assigned_or_suggested:X` returns issues assigned to X, or where X is an owner determined by ownership rules, or where X is a [suspect committer](product/releases/suspect-commits/)

Author: ```suggestion
- `assigned_or_suggested:X` returns issues assigned to X, or where X is an owner determined by ownership rules, or where X is a [suspect committer](/product/releases/suspect-commits/)
```

---
title: Issue States and Triage
sidebar_order: 30
description: "Learn more about issue states and how to triage issues."
---

> 
> An issue can be in one of three states:

Author: ```suggestion
To help organize issues for you and your team, you can take actions such as assigning, resolving, reviewing, or ignoring issues. These actions are referred to as _triaging_.

An issue can be in one of three states:
```

1. Unresolved: The default state for all new issues.
2. Ignored: The issue is ignored permanently or until a condition is met to move it to a state of Unignored. [Issue alerts](/product/alerts-notifications/issue-alerts/) are suppressed for ignored issues.
3. Resolved: The issue is resolved because it was fixed. You can also mark an issue as Resolved if you don't expect it to happen again.

The "For Review" tab, or Review List, of the Issues page displays issues when any of the following happens:

- A new issue is created; it's labeled "New Issue"
- A resolved issue re-occurs; it's labeled "Regression"
- An ignored issue meets its ignore conditions; it's labeled "Unignored"

For-review issues are a subset of all unresolved issues.

## Issue Triage

> Triaging is the act of deciding what action you want to take on an issue. It typically involves one or more of these actions.

Reviewer 2: > To help you organize issues for you and your team you can assign, resolve, review, or ignore an issue. These actions are referred to as "triaging."

Reviewer 2: I think we should move this description to the top. Explain roughly what triaging is, then explain the states and actions.

Author: Can do. Suggest the following edit:

```
To help organize issues for you and your team, you can take actions such as assigning, resolving, reviewing, or ignoring issues. These actions are referred to as _triaging_.
```

### Assign

> You can assign issues to teams or individuals. In a multi-team environment, routing issues to the right people usually starts with assigning it to the right team, which further assigns it to the right member of the team. [Ownership rules](/product/error-monitoring/issue-owners/) allow you to define rules for which teams or individuals should own an issue. You might also see a list of suggested assignees including the owners of suspect commits.

Reviewer 2: > You can assign issues to teams or individuals manually or let Sentry automatically assign issues by setting up [ownership rules](link). This helps you and your team see only the issues that you care about.

Reviewer 2: I know I’m rewriting it above but my real point here is that this is very confusing:

> In a multi-team environment, routing issues to the right people usually starts with assigning it to the right team, which further assigns it to the right member of the team.

Author: I think your initial suggested bit works. 

The sentence you've highlighted as confusing, I took directly from Adi's original draft, but I added a note beneath it because I don't know what it means.

> You can assign issues to teams or individuals. In a multi-team environment, routing issues to the right people usually starts with assigning it to the right team, which further assigns it to the right member of the team. [Ownership rules](/product/error-monitoring/issue-owners/) allow you to define rules for which teams or individuals should own an issue. You might also see a list of suggested assignees including the owners of suspect commits.

Author: ```suggestion
You can assign issues to teams or individuals manually or let [sentry.io](https://sentry.io) automatically assign issues by setting up [ownership rules](/product/error-monitoring/issue-owners/). This helps you and your team see only the issues that you care about. You might also see a list of suggested assignees including the owners of suspect commits.
```

> TO DO Confirm meaning in first sentence; does someone have to manually do this reassigning to a team member after it gets assigned to a team?

Author: ```suggestion
```

> ### Link

Reviewer 2: Can we call this `Linked Issues`?

Author: I suggest "Link Issues" because all of these titles are actions you can take on an issue and we want to maintain parallel structure.

> ### Link

Author: ```suggestion
### Link Issues
```

> Create an issue in an external project management tool like Jira that tracks the Sentry issue, or link an existing one. [Two-way sync](/product/sentry-basics/guides/getting-started/#33-issue-tracking) is available for some tools. Issues linked using [global integrations](/product/integrations/integration-platform/#public-integrations) or [internal integrations](/product/integrations/integration-platform/#internal-integrations) are searchable using the terms `is:linked` and `is:unlinked`.

Reviewer 2: We should have an image here of where that is on the issue details page

Author: ok

> Create an issue in an external project management tool like Jira that tracks the Sentry issue, or link an existing one. [Two-way sync](/product/sentry-basics/guides/getting-started/#33-issue-tracking) is available for some tools. Issues linked using [global integrations](/product/integrations/integration-platform/#public-integrations) or [internal integrations](/product/integrations/integration-platform/#internal-integrations) are searchable using the terms `is:linked` and `is:unlinked`.

Author: ```suggestion
Create an issue in an external project management tool like Jira that tracks the Sentry issue, or link an existing one. [Two-way sync](/product/sentry-basics/guides/getting-started/#33-issue-tracking) is available for some tools. Issues linked using [global integrations](/product/integrations/integration-platform/#public-integrations) or [internal integrations](/product/integrations/integration-platform/#internal-integrations) are searchable using the terms `is:linked` and `is:unlinked`.

TO DO add image here
```

---
title: Issue States and Triage
sidebar_order: 30
description: "Learn more about issue states and how to triage issues."
---

> To help organize issues for you and your team, you can take actions such as assigning, resolving, reviewing, or ignoring issues. These actions are referred to as _triaging_.

Reviewer 3: "reviewing" goes into EA

> To help organize issues for you and your team, you can take actions such as assigning, resolving, reviewing, or ignoring issues. These actions are referred to as _triaging_.

Author: ```suggestion
To help organize issues for you and your team, you can take actions such as assigning, resolving, or ignoring issues. These actions are referred to as _triaging_.
```

An issue can be in one of three states:

1. Unresolved: The default state for all new issues.
2. Ignored: The issue is ignored permanently or until a condition is met to move it to a state of Unignored. [Issue alerts](/product/alerts-notifications/issue-alerts/) are suppressed for ignored issues.
3. Resolved: The issue is resolved because it was fixed. You can also mark an issue as Resolved if you don't expect it to happen again.

> The "For Review" tab, or Review List, of the Issues page displays issues when any of the following happens:

Reviewer 3: goes to EA section

> The "For Review" tab, or Review List, of the Issues page displays issues when any of the following happens:

Author: ```suggestion
```

> - A new issue is created; it's labeled "New Issue"
> - A resolved issue re-occurs; it's labeled "Regression"
> - An ignored issue meets its ignore conditions; it's labeled "Unignored"
> 
> For-review issues are a subset of all unresolved issues.
> 

Author: ```suggestion
```

## Issue Triage

Triaging is the act of deciding what action you want to take on an issue. It typically involves one or more of these actions.

### Assign

You can assign issues to teams or individuals manually or let [sentry.io](https://sentry.io) automatically assign issues by setting up [ownership rules](/product/error-monitoring/issue-owners/). This helps you and your team see only the issues that you care about. You might also see a list of suggested assignees including the owners of suspect commits.

### Link Issues

Create an issue in an external project management tool like Jira that tracks the Sentry issue, or link an existing one. [Two-way sync](/product/sentry-basics/guides/getting-started/#33-issue-tracking) is available for some tools. Issues linked using [global integrations](/product/integrations/integration-platform/#public-integrations) or [internal integrations](/product/integrations/integration-platform/#internal-integrations) are searchable using the terms `is:linked` and `is:unlinked`.

TO DO add image here

> ### Mark Reviewed

Reviewer 3: goes to EA

> ### Mark Reviewed

Author: ```suggestion
```

> Mark the issue as reviewed and [sentry.io](https://sentry.io) removes the reason label (the issue stays in the Unresolved state). Unlike ignoring an issue, this is a way to acknowledge an issue that you intend to fix at some point. This is only available on issues flagged for review)
> 
> Note that resolving or ignoring an issue implicitly marks the issue reviewed.
> 
> To keep your Review List fresh, [sentry.io](https://sentry.io) automatically marks issues as reviewed if they remain untouched in the list for seven days.
> 

Author: ```suggestion
```

### Ignore

Ignore an issue **permanently** (the default action when you press the button) if it's just noise and you'll likely never care about it. Or ignore it until a condition is met (at which point it becomes Unresolved and flagged for review). You can also set an issue to Ignore if you're working on the issue if you don't want to keep getting alerts.

### Delete

Delete an issue to remove it from the system. A new issue is created if a new instance of the issue (a new event) occurs.

### Delete and Discard

Delete and discard an issue to remove it from the system. A new issue is not created even if a new instance of the issue (a new event) occurs.

### Reprocess

[Reprocessing](/product/error-monitoring/reprocessing/) allows you to apply source maps and debug information files to error events that have already occurred.

<Note>

The Auto Resolve setting defines the amount of time an issue can go without being seen again (that is, no new events on that issue) before it's automatically resolved. To check if this has been defined for a project, go to **[Project] > Settings > General Settings** and check the Event Settings section.

</Note>

> While you can work to have zero unresolved issues ("Inbox Zero") by resolving or ignoring all issues, this usually isn't feasible for any non-trivial production application. However, it is feasible to have zero for-review issues by triaging the Review List. We recommend triaging these issues once a day to keep your issues manageable.

Reviewer 3: goes into EA

> While you can work to have zero unresolved issues ("Inbox Zero") by resolving or ignoring all issues, this usually isn't feasible for any non-trivial production application. However, it is feasible to have zero for-review issues by triaging the Review List. We recommend triaging these issues once a day to keep your issues manageable.

Author: ```suggestion
## Issues Page Tabs

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.

If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

The Issues page is organized into tabs, each corresponding to a filtered list of issues:

- All Unresolved (`is:unresolved`): All unresolved issues, including issues in the "For Review" tab.
- For Review (`is:unresolved is:for_review`): Issues that need to be reviewed; for-review issues are a subset of all unresolved issues.
- Ignored (`is:ignored`): All ignored issues.
- Saved Searches: Select from a set of [premade](/product/sentry-basics/search/#premade-searches) and custom [saved searches](/product/sentry-basics/search/#organization-wide-saved-searches). The name of the tab changes based on your selection.

These different lists help you with the triaging process. 

The "For Review" tab, or Review List, of the Issues page displays issues when any of the following happens:

- A new issue is created; it's labeled "New Issue"
- A resolved issue re-occurs; it's labeled "Regression"
- An ignored issue meets its ignore conditions; it's labeled "Unignored"

You can mark issues in this tab as reviewed and [sentry.io](https://sentry.io) removes the reason label (the issue stays in the Unresolved state). Unlike ignoring an issue, this is a way to acknowledge an issue that you intend to fix at some point. This is only available on issues flagged for review.

Note that resolving or ignoring an issue implicitly marks the issue reviewed.

To keep your Review List fresh, [sentry.io](https://sentry.io) automatically marks issues as reviewed if they remain untouched in the list for seven days.

While you can work to have zero unresolved issues ("Inbox Zero") by resolving or ignoring all issues, this usually isn't feasible for any non-trivial production application. However, it is feasible to have zero for-review issues by triaging the Review List. We recommend triaging these issues once a day to keep your issues manageable.
```
